ValidationLibrary/FixableRuleBase.cs

```diff
@@ -115,10 +115,10 @@ protected async Task<IReadOnlyList<RepositoryContent>> GetContents(IGitHubClient
                  * Octokit.NotFoundException. This same thing would probably be throw if the whole repository
                  * was missing, but we don't care for that case (no point to validate if repository doesn't exist.)
                  */
-                _logger.LogWarning(exception, "Rule {ruleClass} / {ruleName}, Repository {repositoryName} caused {exceptionClass}. This may be a new repository, but if this persists, repository should be removed.",
```

Comment:
Just fix `nameof(HasNewestPtcsJenkinsLibRule)` with `typeof(T).Name`, that was a bug in logging.

`{ruleClass}` and `{ruleName}` have a specific usage here (log filtering with Application) so those should be here

Reply:
Done.